tests/unit/backend/corpora/lambdas/dataset_submissions/test_dataset_submissions.py

```diff
@@ -54,15 +54,18 @@ def test__non_owner__raises_error(self, mock_get_dataset_info):
         self._test_types()
 
     @patch("backend.corpora.dataset_submissions.app.get_dataset_info")
-    @patch("backend.corpora.dataset_submissions.app.upload")
-    def test__owner__upload(self, mock_upload: Mock, mock_get_dataset_info: Mock):
+    def test__upload__OK(self, mock_get_dataset_info: Mock):
```

Comment:
Ha, I forgot to add the `Mock` type in tests I wrote previously.

Reply:
not required but always helpful to have.